src/test/java/teammates/test/cases/browsertests/FeedbackConstSumOptionQuestionUiTest.java

```diff
@@ -251,6 +257,82 @@ public void testDeleteQuestionAction() {
         assertNull(BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 1));
     }
 
+    /**
+     * Tests that distribute points among options (new and existing) can be reordered using drag and drop mechanism.
+     * @throws Exception when option is not draggable
+     */
+    private void testReorderOptions() throws Exception {
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionTypeAndWaitForNewQuestionPanelReady("CONSTSUM_OPTION");
+
+        feedbackEditPage.fillQuestionTextBoxForNewQuestion("Test question text");
+        feedbackEditPage.fillQuestionDescriptionForNewQuestion("more details");
+        feedbackEditPage.clickAddMoreConstSumOptionLinkForNewQuestion();
+        feedbackEditPage.clickAddMoreConstSumOptionLinkForNewQuestion();
```

Comment:
Any reason for doing this twice?

Reply:
well its simply adding two new options in the grid i.e. clicking `Add More Options` twice. I did this to test if users can add multiple options and sort them in one go before saving